public/javascript/main.js

```diff
@@ -376,5 +376,62 @@ $(function(){
 
           $(this).data('chart', chart);
       }
-  })
+  });
+
+  $(".js-download-with-term").on("click", function(e){
+    var term = $(e.target).data("term");
+    var house = $(e.target).data("house");
+    if (term && house) {
+        // inject query vars into URL before the target anchor
+        e.preventDefault();
+        location = $(e.target).attr('href')
+          .replace('#', "?" + $.param({term: term, house: house}) + "#");
+    }
+  });
+
+  if (typeof pageInitFunction === 'function') {
+      pageInitFunction();
+  }
+
 });
+
+// collapse_displayed_items:
+// If there are too many items being displayed in the list, hide
+// all but a few, making sure the target element (if there is one)
+// is included in those that are not hidden. Add a button for
+// revealing the hidden items.
+// Note this is used on the download page, where (currently) there
+// may be more than one such list (e.g., one for each legislature).
+
+function collapse_displayed_items(
+    $list_of_elements,   // list of elements that need to be collapsed
```

Comment:
@zarino as I'm working through making these changes, I just want to comment on this one because it's the only one I disagree with: I think you're a bit cruel to say that the result of `$('#terms-blah li')` isn't a list in any meaningful sense of the word; I think it is, because the "jQuery object" is effectively (more or less their words) "an ordered set of elements"... which is pretty much the sense of the word "list" no? The sense of it here is that the function accepts such a list, one of which is the target to be highlighted. 

But... needing to revert to such pedantry suggests that it makes much more sense for this function to stop being so general, *admit* it's always going to be a jQuery object containing the parent `ul` element, and take it from there. Then all my talk of `items` where I'm _pretending_ they might not be `li` elements goes away. So good spot after all ;-)